src/gui/accountstate.h

```diff
@@ -161,6 +163,8 @@ class AccountState : public QObject, public QSharedData
     ///Asks for user credentials
     void handleInvalidCredentials();
 
+    PushNotifications *pushNotifications() const;
```

Comment:
We probably need a signal to also claim on the account when the push notifications get available or when they're not anymore (think auth error, network error, etc.). This signal would then need to be handled by FolderMan to decide to move a folder to/from the polling timer or to/from the corresponding PushNotification object.